### ISSUE TITLE
Multi Object Detect Is Wrong!

### ISSUE THREAD

**Issue by wwzh2015** [AIWintermuteAI/aXeleRate#60]
**Describe the bug**
Multi Object Detect Is Wrong

**To Reproduce**
Train multi classes object detector by pascal_20_detector.json.

**Expected behavior**
If I just have only one class object that‘s mAp calculate is true, However when multi class objects(class>1) that's mAp is 0 or very low!
and object detect have nothing result. what's happened?

**Screenshots**
https://ftp.bmp.ovh/imgs/2021/07/9a67e2599fcda7c9.png
https://ftp.bmp.ovh/imgs/2021/07/a9db47136bcfbd7d.jpg

**Environment (please complete the following information):**
- Local: win10 anaconda3 tensorflow1.15.5 kreas 2.3.1 axelerate 0.6.0 Mobilenet5_0/7_5/1_0 or Tiny Yolo

**Additional context**
Why the four classes ap also is 0?
battery 0.0000
cardboard 0.0000
glass 0.0000
metal 0.0000
mAP: 0.0000
mAP did not improve from 0.
Epoch 42/100
5/5 [==============================] - 1s 141ms/step - loss: 1.0767 - val_loss: 0.7849

**Comment by AIWintermuteAI** [comment]
Thanks for filling the from.
From what I see in your screenshots, you can get 0.1 mAP, which is not zero, but very low. 

Judging by this line 
5/5 [==============================] - 1s 141ms/step
it only takes 1 second for full pass over dataset - so either you have a very powerful machine or your dataset is extremely small. Remember that at very least 250-500 instances is required per class to achieve satisfactory results, if you use transfer learning. Much more is needed if you train from scratch.

This is all I can say without looking at
a)config
b)dataset

**Comment by wwzh2015** [comment]
Thanks
